_posts/2021-09-01-Fortran-Newsletter-September-2021.md

```diff
@@ -56,7 +56,34 @@ environment in which new open source Fortran projects are created and published
 
 ### Flang
 
-TODO @AlexisPerry
+Recent development updates:
+
+* New Driver and CMake integration:
+    * The new driver has replaced the old, hence f18 has been deleted. 
+    * flang-new (aka. the new driver) now drives the flang bash script before an external compiler is called.
+    * Code-generation work is ongoing.
+    * Work is now proceeding to enable CMake to recognize the compiler and set the appropriate options for build configurations.
+* FIR (Fortran IR - a dialect of MLIR):
+    * Fortran 95 lowering and runtime support is nearing completion, particularly of intrinsics
+    * Code upstreaming will begin again in earnest once F95 is deemed complete
+* OpenMP
+    * Nesting of region semantic checks
+    * enter_data MLIR to LLVM IR lowering
+    * Semantic checks for allocate directive
+    * Lowering for various modifiers for the schedule clause
+    * Pretty printer and parser for omp.target operation
+    * Semantic checks for linear, nested barrier, allocate directive
+    * Progress with firstprivate, critical, collapse, ordered, reduction
+* Lift -Werror checks into local functions
+* Document the flang wrapper script
+* Fix the extent calculation when upper bounds are less than lower bounds
+* Fix list-directed plural repeated null values at end of record
+* Fix build failure on MacOS involving std::clock_t
+* Fix error messages on Windows.
+* Disable Plugins in out-of-tree builds
+* Correct off-by-one error in SET_EXPONENT
+
+Call notes are recorded [here](https://docs.google.com/document/d/10T-S2J3GrahpG4Ooif93NSTz2zBW0MQc_RlwHi0-afY).
```

Comment:
This link requires access, so it is not public. I think I would not post it here, since people (outside the Flang development team) will not be able to click on it.

Reply:
I agree with @certik. @AlexisPerry is it feasible to make the doc publicly readable, or should we just remove the link?

Reply:
Fair enough.  People can always request access, which is why I included it, but since that requires an extra step perhaps it's not for the best.  I'll remove it.

Reply:
If anybody can request access then you can write there how they can request access and mention that. Perhaps something like:

> Flang call notes are recorded and available upon request (please contact Alexis Perry at XXX).

And if they contact you, you can send them the link.

Reply:
I think being able to request access is okay, so I recommend keeping it, if @certik is fine with it. Perhaps rewording as: "Call notes are recorded and available by request here...".